### ISSUE TITLE
OS return value: -4?

### ISSUE THREAD

**Issue by bpeacock44** [ablab/spades#1087]
### Description of bug

Hello, 

I am attempting to do read error correction on a set of paired end reads but I keep getting OS return value: -4. I don't see any indication in the log of a specific issue. Please help me troubleshoot. Much appreciated!

Thank you,

-Beth Peacock  

### spades.log

[spades.log](https://github.com/ablab/spades/files/10797057/spades.log)

### params.txt

[params.txt](https://github.com/ablab/spades/files/10797059/params.txt)

### SPAdes version

SPAdes 3.15.3

### Operating System

Linux-3.10.0-1160.62.1.el7.x86_64-x86_64-with-centos-7.9.2009-Core

### Python Version

2.7.5

### Method of SPAdes installation

Spack

### No errors reported in spades.log

- [X] Yes

**Comment by asl** [comment]
Will you please run `spades.py --test` on your system?

**Comment by bpeacock44** [comment]
Yes. It says the test ran correctly (though it did have a warning — * 0:00:04.454 1M / 150M WARN General (launcher.cpp : 178) Your data seems to have high uniform coverage depth. It is strongly recommended to use --isolate option.)

I should have mentioned - I have run this command before and it has completed successfully but the data file I am attempting to run it on now is much larger than the others had been. That is the main difference.

**Comment by asl** [comment]
Well, likely something changed on your system / prevented SPAdes from run. crash effectively occurs during the startup of the binary. I would suggest you to contact your system administrator.
